packages/spark-core/settings/_mappings.scss

```diff
@@ -0,0 +1,144 @@
+$btn-hover-text-color: $sprk-btn-hover-text-color;
+$btn-padding: $sprk-btn-padding;
+$btn-text-color: $sprk-btn-text-color;
+$btn-border-width: $sprk-btn-border-width;
+$btn-hover-background-color: $sprk-btn-hover-background-color;
+$btn-text-color: $sprk-btn-text-color;
+$btn-hover-text-color: $sprk-btn-hover-text-color;
+$btn-border-style: $sprk-btn-border-style;
+
+//
+// Disabled
+///
+$btn-disabled-opacity: $sprk-btn-disabled-opacity;
+$btn-disabled-background-color: $sprk-btn-disabled-background-color;
+$btn-disabled-text-color: $sprk-btn-disabled-text-color;
+$btn-disabled-border-width: $sprk-btn-disabled-border-width;
+$btn-disabled-border-color: $sprk-btn-disabled-border-color;
+$btn-disabled-background-color: $sprk-btn-disabled-background-color;
+$btn-disabled-border-color: $sprk-btn-disabled-border-color;
+$btn-disabled-text-color: $sprk-btn-disabled-text-color;
+
+//
+// Layout
+///
+$centered-column-width: $sprk-centered-column-width;
+$fieldset-padding: $sprk-fieldset-padding;
+
+//
+// Lists
+///
+$list-margin-left: $sprk-list-margin-left;
+
+//
+// Horizontal List
+///
+$horizontal-list-spacing: $sprk-horizontal-list-spacing;
+$horizontal-list-spacing-tiny: $sprk-horizontal-list-spacing-tiny;
+$horizontal-list-spacing-small: $sprk-horizontal-list-spacing-small;
+$horizontal-list-spacing-medium: $sprk-horizontal-list-spacing-medium;
+$horizontal-list-spacing-large: $sprk-horizontal-list-spacing-large;
+$horizontal-list-spacing-huge: $sprk-horizontal-list-spacing-huge;
+
+//
+// Inputs
+///
+$input-container-margin-bottom: $sprk-input-container-margin-bottom;
+$input-disabled-background-color: $sprk-input-disabled-background-color;
+$input-disabled-border-color: $sprk-input-disabled-border-color;
+$input-disabled-text-color: $sprk-input-disabled-text-color;
+$input-readonly-border-color: $sprk-input-readonly-border-color;
+$input-readonly-text-color: $sprk-input-readonly-text-color;
+$input-container-margin-bottom: $sprk-input-container-margin-bottom;
+$input-disabled-background-color: $sprk-input-disabled-background-color;
+$input-disabled-border-color: $sprk-input-disabled-border-color;
+$input-disabled-text-color: $sprk-input-disabled-text-color;
+$input-readonly-border-color: $sprk-input-readonly-border-color;
+$input-readonly-text-color: $sprk-input-readonly-text-color;
+$text-input-border-radius: $sprk-text-input-border-radius;
+$text-input-border-color: $sprk-text-input-border-color;
+$text-input-border-width: $sprk-text-input-border-width;
+$text-input-padding: $sprk-text-input-padding;
+
+//
+// Media
+///
+$media-gutter-tiny: $sprk-media-gutter-tiny;
+$media-gutter-small: $sprk-media-gutter-small;
+$media-gutter: $sprk-media-gutter;
+$media-gutter-large: $sprk-media-gutter-large;
+$media-gutter-huge: $sprk-media-gutter-huge;
+
+//
+// Reset
+///
+$selection-bg-color: $sprk-selection-bg-color;
+
+//
+// Stack
+///
+$split-breakpoint-xs: $sprk-split-breakpoint-xs;
```

Comment:
These should be put in the settings file. They are in scales but we do use them. I built the stack object inside of Scales but then decided it was best to be in Spark. So, the Stack object is in both places right now but Scales will be removing it as it hasn't been maintained/updated.